geodiff/CMakeLists.txt

```diff
@@ -7,11 +7,15 @@ SET(CMAKE_CXX_VISIBILITY_PRESET hidden)
 SET(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
 SET(CMAKE_CXX_STANDARD 11)
 
+# set path to additional CMake modules
+SET(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
+
 SET(ENABLE_TESTS TRUE CACHE BOOL "Build tests?")
 SET(ENABLE_COVERAGE FALSE CACHE BOOL "Enable GCOV code coverage?")
 SET(BUILD_TOOLS TRUE CACHE BOOL "Build tool executables?")
-SET(BUILD_STATIC_LIBS FALSE CACHE BOOL "Build only static libraries?")
+SET(BUILD_STATIC FALSE CACHE BOOL "Build static libraries?")
```

Comment:
can we keep the existing define BUILD_STATIC_LIBS (even if it changes the meaning). otherwise you need to modify input-sdk for ios at least

Reply:
To be addressed in https://github.com/lutraconsulting/input-sdk/issues/22